codalab/lib/bundle_cli.py

```diff
@@ -775,17 +788,20 @@ def do_help_command(self, args):
         help='Show current client status.'
     )
     def do_status_command(self, args):
+        client, worksheet_uuid = self.manager.get_current_worksheet_uuid()
+        worksheet_info = client.fetch('worksheets', worksheet_uuid)
+
         if not self.headless:
             print >>self.stdout, "codalab_home: %s" % self.manager.codalab_home
             print >>self.stdout, "session: %s" % self.manager.session_name()
             address = self.manager.session()['address']
+            print >>self.stdout, "client_version: %s" % CODALAB_VERSION
```

Comment:
Still would prefer to have `cl -v` and `cl --version` work instead of having it here, since that's what people are used to typing.